src/App.tsx

```diff
@@ -152,7 +177,7 @@ export function App(): JSX.Element {
     codeElement.textContent = code || " ";
     preRef.current.appendChild(codeElement);
     prism.highlightAllUnder(preRef.current);
-    addInlineStyles(preRef.current);
+    inlineStyles(preRef.current);
```

Comment:
I was about to say "Hmm, I like this better when it has a verb", but then I realized "inline" **is** a verb... I don't love the new name, but I also can't think of a better one right now